modules/rgbd/src/hash_tsdf.cpp

```diff
@@ -540,7 +579,7 @@ struct PushNormals
         if (!isNaN(p))
         {
             Point3f voxelPoint = invPose * p;
-            n                  = volume.pose.rotation() * volume.getNormalVoxel(voxelPoint);
+            n                  = volume.pose * volume.getNormalVoxel(voxelPoint);
```

Comment:
there should be `rotation()`, won't work without it

Reply:
done